### DISCUSSION TITLE
cannot get the ssd temperature - KINGSTON OM3PDP3512B-AB

### DISCUSSION THREAD

**Discussion by real1993** [LibreHardwareMonitor/LibreHardwareMonitor#1167] [General]
Cannot get the ssd temperature - KINGSTON OM3PDP3512B-AB
diskinfo can, how can i get that in this applicant, thanks.

**Comment by Eta-Beta** [comment]
Hi, I'm not an expert but is the SSD disk a classic 2.5" inch Sata or one of the new M2s?, because in the first case they practically never heat up while the M2s are a completely different thing.
